README.md

```diff
@@ -132,8 +132,19 @@ go run examples/dial/psk/main.go
   openssl s_client -dtls1_2 -connect 127.0.0.1:4444 -psk abc123 -cipher PSK-AES128-CCM8
 ```
 
+### Roadmap
+The library is used as a part of our WebRTC implementation. Please refer to that [roadmap](https://github.com/pion/webrtc/issues/9) to track our major milestones.
```

Comment:
I'd like to remove this. The DTLS library is used well beyond WebRTC and we don't plan future work on DTLS in the webrtc project.

Reply:
Alright. I removed the section.

However, regarding your second point: I still see DTLS as an important component for WebRTC and there are definitely others how contribute to this repo with having better interoperability of Pion's WebRTC/TURN implementation in mind.